apps/condo/domains/organization/integrations/sbbol/sync/index.js

```
@@ -104,16 +106,53 @@
+        throw error
+    }
+    let organizationSyncResult
+    try {
+        organizationSyncResult = await syncOrganization({ context, user, userData, organizationInfo, dvSenderFields })
+    } catch (err) {
+        logger.error({
+            msg: 'Failed to sync organization',
+            err,
+            data: { userData, userInfo, organizationInfo },
+        })
+        throw error
```

ESLint

[error] 'error' is not defined.

(no-undef)